### ISSUE TITLE
UI present bento boxes as cards

### ISSUE THREAD

**Issue by StudioZut** [gwu-libraries/obento#275]
Enhance design of bento boxes to "box" them out as cards, with the same layout but clearer distinction between sections (vs just the black line/horizontal rule)

**Comment by StudioZut** [comment]
Example of how this might look. The box titles (Articles, Databases etc) would likely be moved inside the 'card' itself. 
![screen shot 2015-01-22 at 9 53 32 am](https://cloud.githubusercontent.com/assets/1031373/5857822/89b3f0be-a21c-11e4-9c69-fd9e07a33f90.png)

**Comment by StudioZut** [comment]
I will build a few variants with shading and colors to present to the team.

**Comment by StudioZut** [comment]
Cleaning up the html structure first.

**Comment by StudioZut** [comment]
Screenshot of the card design after the html/bootstrap cleanup (still some adjustments to do, like rounding the gray search box to match the cards)
![screen shot 2016-01-08 at 4 51 57 pm](https://cloud.githubusercontent.com/assets/1031373/12210527/aa264b6e-b628-11e5-9e59-43b513cf195c.png)
:

**Comment by StudioZut** [comment]
And in handheld narrow view (iphone6 emulation):
![screen shot 2016-01-08 at 4 50 19 pm](https://cloud.githubusercontent.com/assets/1031373/12210552/cd6797e0-b628-11e5-8ae2-91447f4b2a2a.png)

**Comment by kerchner** [comment]
@StudioZut nice.  Can the CSS be enhanced to give each box a shadow, to create the sense that they are floating?

**Comment by StudioZut** [comment]
@kerchner I'll mock that up (drop shadow) and add screenshots

**Comment by StudioZut** [comment]
@kerchner here's a version with box-shadow (box-shadow: 2px 2px 10px #ccc;)
![screen shot 2016-01-13 at 4 41 07 pm](https://cloud.githubusercontent.com/assets/1031373/12308734/dd4a9352-ba14-11e5-8e35-dd72b99c29ea.png)

**Comment by kerchner** [comment]
@StudioZut I like it.  How does it look if you also give the "cards" a slightly different background color vs. the white page background?

**Comment by StudioZut** [comment]
@kerchner making me work! This is with "buff" as background: 
![screen shot 2016-01-19 at 12 11 30 pm](https://cloud.githubusercontent.com/assets/1031373/12426091/e2d47938-bea5-11e5-87eb-d11e3b901ddb.png)

**Comment by StudioZut** [comment]
This is a new 'button' look for the Jumpto links (which also fixes #278)
![screen shot 2016-01-19 at 1 52 28 pm](https://cloud.githubusercontent.com/assets/1031373/12428712/46b8f25e-beb4-11e5-8c13-83fe29654d35.png)

**Comment by kerchner** [comment]
This will be a nice improvement.  My preference is for the `#eee` background, as it provides a 3-d look but still maintains a high contrast.  @lwrubel , @adityadharne what do you think?

**Comment by StudioZut** [comment]
Take a look on test http://test.library.gwu.edu/search-all?query=moby (and resize the window to see how it responds)

**Comment by kerchner** [comment]
@StudioZut responsiveness works flawlessly.  Is there a reason that the blue color of the buttons doesn't match the blue used on the library site (for example, in the "Search" and "Go" buttons and header background?

**Comment by lwrubel** [comment]
I prefer the simple white on white look. Good contrast for text and more
consistent with other card-using websites. If anything the background is
the color that changes in card sites.

**Comment by StudioZut** [comment]
@kerchner good catch, I changed it to the darker blue.  I also updated the "search" button above it to match better. 
@lwrubel I agree, here it is in "white on white" so we can take a look.
http://test.library.gwu.edu/search-all?query=books

**Comment by adityadharne** [comment]
I think that the white on white looks better.

**Comment by kerchner** [comment]
Let's go with white on white then.   The buttons now match, although I noticed:
- The "Got feedback?" button is still the other blue, except when you mouse over it; then it changes to the blue that matches the others.
- The font size in the buttons that appear when the width is narrow (i.e. mobile view) is somewhat larger than the other buttons.  Is this by design?

**Comment by StudioZut** [comment]
I resized the "results by" jump buttons (this was a failed effort to get them to distribute evenly across a typical phone screen)
I changed the "got feedback" color from the "link blue" that buttons default to (#2971a7). It's now the blue of the other buttons on this page (#4282B1). 

I also adjusted the "search" box to match the "card" design of the results.

http://test.library.gwu.edu/search-all?query=ziggy
